src/main/webapp/view/transcript/transcript-seeker.js

```diff
@@ -50,15 +52,20 @@ export default class TranscriptSeeker {
    * of the container.
    */
   static scrollToTopOfTranscript(transcriptLine) {
+    if (!TranscriptArea.transcriptScrollContainer().autoScrollIsActive()) {
+      return;
+    }
     const transcriptContainer =
         document.getElementById(TranscriptSeeker.#TRANSCRIPT_CONTAINER);
     const ulElementOffset = transcriptLine.parentElement.offsetTop;
     transcriptContainer.scrollTop = transcriptLine.offsetTop - ulElementOffset;
+    TranscriptArea.transcriptScrollContainer().browserScrolled = true;
```

Comment:
Yeah this could all be brought into the scroll container, (switching the `.getElementById` with `this`)

Reply:
done!